hedera-node/hedera-token-service-impl/src/main/java/com/hedera/node/app/service/token/impl/handlers/TokenAirdropHandler.java

```diff
@@ -132,6 +135,8 @@ public void handle(@NonNull final HandleContext context) throws HandleException
         assessAndChargeCustomFee(context, convertedOp);
```

Comment:
We should have a refactor task to break this method (`.handle`) up into smaller pieces.  Its doing a lot of work.  It's all closely related, and _seems_ straightforward.  But the complexity is indicated by the fact that when it was written only _two_ unit tests were provided which does not seem adequate in any way, but it's my inference that unit testing it properly, as it stands, was too daunting.

Reply:
I created an issue for that https://github.com/hashgraph/hedera-services/issues/14505 but for some reason, it got deleted 🤔 I will ask in the services channel why.